src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/Compiler/RegisterGlobalSecurityEventListenersPassTest.php

```diff
@@ -146,7 +155,14 @@ private function assertListeners(array $expectedListeners, string $dispatcherId
         }
 
         $foundListeners = array_uintersect($expectedListeners, $actualListeners, function (array $a, array $b) {
-            return $a === $b;
+            // The comparison function must return an integer less than, equal to, or greater than zero if the first
+            // argument is considered to be respectively less than, equal to, or greater than the second.
+            if ($a === $b) {
+                return 0;
+            }
+
+            // PHP internally sorts all the arrays first, so returning proper 1 / -1 values is crucial
+            return $a <=> $b;
```

Comment:
Wow, didn't know this. This can be simplified by `return $a <=> $b;` as the spaceship operator takes care of equal as well.